scarb/tests/e2e/new_and_init.rs

```diff
@@ -146,6 +147,34 @@ fn new_existing() {
         "#});
 }
 
+#[test]
+fn new_no_abs_path_warning() {
```

Comment:
```suggestion
fn issue_148() {
```

the issue description tells much more than this test name